examples/App.config

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <!--*************************************************************************************-->
+  <!-- Log4Net configuration sections below are needed only if Log4NetLoggerService is used -->
+  <!--**************************************************************************************-->
+  <configSections>
+    <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, log4net" />
+  </configSections>
+
+  <log4net>
+    <root>
+      <level value="DEBUG" />
+      <appender-ref ref="ConsoleAppender" />
+      <appender-ref ref="LogFileAppender" />
+    </root>
+    <appender name="ConsoleAppender" type="log4net.Appender.ConsoleAppender">
+      <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="[%date] [%thread] [%-5level] [%logger] - %message%newline" />
+      </layout>
+    </appender>
+    <appender name="LogFileAppender" type="log4net.Appender.RollingFileAppender">
+      <file type="log4net.Util.PatternString">
+        <conversionPattern value="%env{TEMP}\\SparkCLRLogs\\SparkCLRExamples_%env{COMPUTERNAME}[%processid].log" />
+      </file>
+      <param name="AppendToFile" value="true" />
+      <param name="MaxSizeRollBackups" value="2000" />
+      <param name="MaxFileSize" value="51200000" />
+      <param name="StaticLogFileName" value="false" />
+      <param name="DatePattern" value=".yyyy_MM_dd_hh" />
+      <param name="RollingStyle" value="Composite" />
+      <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="[%date] [%thread] [%-5level] [%logger] - %message%newline" />
+      </layout>
+    </appender>
+  </log4net>
+  
+  <appSettings>
+    <!--********************************************************************************************************-->
+    <!--** Uncomment the following settings to run Spark driver executable in **local** or **debug** modes ** -->
+    <!--** In debug mode, the driver is not launched by CSharpRunner but launched from VS or command prompt not configured for SparkCLR ** -->
+    <!--** CSharpBackend should be launched in debug mode as well and the port number from that should be used below ** -->
+    <!--** Command to launch CSharpBackend in debug mode is "sparkclr-submit.cmd debug" ** -->
+    <!--********************************************************************************************************-->
+    
+    <!--
+    <add key="CSharpWorkerPath" value="C:\SparkCLR\csharp\Examples\bin\Debug\CSharpWorker.exe"/>    
+    <add key="CSharpBackendPortNumber" value="0"/>
```

Comment:
Is this stuff needed? 

I suggest might be clearer if Examples code targets what "normal" end-user developer will do, rather than any special debug that core SparkCLR dev team might do.

Goal for Examples should really be: "Just works by pressing `F5` in Visual Studio"